neo/Prompt/Utils.py

```diff
@@ -322,15 +322,22 @@ def verify_params(ptype, param):
     elif ptype == ContractParameterType.ByteArray:
         if isinstance(param, str) and len(param) == 34 and param[0] == 'A':
             return Helper.AddrStrToScriptHash(param).Data, False
-        res = eval(param, {"__builtins__": {'bytearray': bytearray, 'bytes': bytes}}, {})
-        if isinstance(res, bytes):
-            return bytearray(res), False
-        return res, False
+        try:
+            res = eval(param, {"__builtins__": {'bytearray': bytearray, 'bytes': bytes}}, {})
+            if isinstance(res, bytes):
+                return bytearray(res), False
+            return res, False
+        except Exception:
+            pass
+        raise Exception("Please provide a bytearray or bytes object")
 
     elif ptype == ContractParameterType.Array:
-        res = eval(param)
-        if isinstance(res, list):
-            return res, False
+        try:
+            res = eval(param)
+            if isinstance(res, list):
+                return res, False
+        except Exception:
+            pass
```

Comment:
This pass is necessary in case there is no exception thrown but `eval` does not evaluate a `list` object.